docs/docs/producers.md

```diff
@@ -125,6 +125,8 @@ You will need to obtain an IAM user that has the following permissions for the s
 - "kinesis:PutRecord"
 - "kinesis:PutRecords"
 - "kinesis:DescribeStream"
+
+Additionally, the producer will need to be able to produce CloudWatch metrics which requires the following permission:
```

Comment:
you did a a great writeup in the issue, but I think this sentence needs to be clearer -- that permission needs to be applied to "resource: *" or whatever.

Reply:
@osheroff thank you for the feedback. Please see the updated version. I also decided to add the entire policy document as an example. This way it may be even easier to understand what the required IAM policy is.